catalogue/aarch64-ETS2/README.md

```diff
@@ -0,0 +1,73 @@
+aarch64-ETS2: Catalogue with the semantics of Armv8-A VMSA
+============================================================
+
+Includes tests that exercise the semantics of the Virtual Memory
+System Architecture. Most of the tests in this catalogue, concurrently
```

Comment:
I would drop this comma - also why "most of the tests"? not all of them?

Reply:
There are some tests which show UDF, or the ordering effects of DC, IC.

Reply:
Sorry I didn't realise this was the ETS2 catalogue. The test in this catalogue are the same as the VMSA, so the same explanation applies here.